frontend/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -9,20 +10,41 @@ export const authOptions: NextAuthOptions = {
       issuer: process.env.OIDC_AUTHORITY
     })
   ],
+  pages: {
+    error: '/web/v2/auth/error'
+  },
   callbacks: {
     async redirect({ baseUrl }) {
       return `${baseUrl}/web/v2/`
     },
     async session({ session, token }) {
+      const headers = {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token.accessToken}`
+      }
+
+      const response = await fetch(`${process.env.API_BASE}/v1/users/me`, { headers: { ...headers } })
+      const result = await response.json()
```

Comment:
we might not need this here, we can use `token.user` since we already fetched it on the `jwt` method:

![image](https://github.com/Igalia/phpreport/assets/20492786/3ceb178f-ebae-4cbc-9008-1ced16b61ee2)

Reply:
I think you are right. I had initially added the fetch to the session callback and was not getting the data at the right time and must have forgotten to remove it after I had it working. Thanks for the catch.